data/names.txt

```diff
@@ -107,6 +107,17 @@ phrase "adjectives"
 		"Steely"
 		"Sanguine"
 		"Sunny"
+		"Chiseled"
+		"Lightning"
+		"Storm"
```

Comment:
I don't think "Lightning" and "Storm" work as adjectives, but "Stormy" would. That is, "Storm Richard" sounds weird, but "Stormy Richard" would work.